RadarSDK/RadarInitializeOptions.m

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
we need some to dict and from dict helper methods for x-platform compatibility.

Reply:
I shall add them 👍

Reply:
hmm technically the native side of the x platform sdk can just use the native init method... maybe we can get away with not having any? Thoughts @lmeier

Reply:
Hmmm, first instinct is that then the x-platforms are essentially rewriting the `toDict` themselves in the init -- probably nicer to follow the normal abstraction pattern we have established an indeed have both `fromDict` and `toDict` and use those

Reply:
we have from/to dict now